SayTheirNames/Source/View/ImageWithBlurView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ImageWithBlurView.swift
+//  SayTheirNames
+//
+//  Created by Leonard Chen on 6/5/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class ImageWithBlurView: UIView {
```

Comment:
`BlurImageView` or `BlurredImageView` 

Personal preference to not have `with` in class names.

Reply:
My point is to make sure the name can identify this view has a blurred image as background. Doesn't feel `BlurImageView` or `BlurredImageView` make it clear.
`ImageWithBlurView` is not a good name, maybe we can come up with some other ideas.